katello.te

```diff
@@ -53,6 +54,18 @@ miscfiles_read_certs(websockify_t)
 
 ######################################
 #
+# Pulp
+#
+
+# Katello installer configures Celery to listen on port 5000
+ifdef(`distro_rhel6', `
```

Comment:
I did - and it didn't work :-) I had to remove a bunch of duplicate resources as well (elastic search, tasks, etc).

I'll retry with the latest changes